qa/workunits/cephtool/test.sh

```diff
@@ -758,6 +758,7 @@ function test_mon_misc()
 
   ceph mgr stat
   ceph mgr dump
+  ceph mgr dump | jq .active_clients[0] | grep 'name'
```

Comment:
Invoking `grep` just for the exit code is unnecessary here:
```suggestion
  ceph mgr dump | jq -e '.active_clients[0].name'
```